packages/sources/layer2-sequencer-health/src/config/index.ts

```diff
@@ -95,36 +95,53 @@ export const CHAIN_DELTA: Record<Networks, number> = {
 }
 
 const DEFAULT_METIS_HEALTH_ENDPOINT = 'https://andromeda-healthy.metisdevops.link/health'
-export const HEALTH_ENDPOINTS: Record<
+const DEFAULT_SCROLL_HEALTH_ENDPOINT = 'https://venus.scroll.io/v1/sequencer/status'
+
+export type HeathEndpoints = Record<
   Networks,
-  { endpoint: string | undefined; responsePath: string[] }
-> = {
+  {
+    endpoint: string | undefined
+    responsePath: string[]
+    processResponse: (data: unknown) => boolean | undefined
+  }
+>
+
+export const HEALTH_ENDPOINTS: HeathEndpoints = {
   [Networks.Arbitrum]: {
     endpoint: util.getEnv('ARBITRUM_HEALTH_ENDPOINT'),
     responsePath: [],
+    processResponse: () => undefined,
```

Comment:
settled on leaving it as `undefined` for networks without a healthcheck to separate from failing checks